docs/UserGuide.md

```diff
@@ -127,7 +128,7 @@ Format: `findt KEYWORD [MORE_KEYWORDS]`
 - The order of the keywords matters. e.g. `drink water` will not match `water drink`
 - Only full keyword(s) will be accepted. e.g. `drink` will match `drink`; `ink` will not match `drink`
 - All keyword(s) have to be matched for task to be returned.
-- If a word is contained within a parantheses without spaces, the keyword has to contain the parantheses as well. 
```

Comment:
@yumengtan nice try with the spelling! 😵‍💫

Reply:
@putaojuice great catch!

Reply:
> @yumengtan nice try with the spelling! 😵‍💫

oops! 🙊  @putaojuice thanks for catching my mistake! 💯